src/Tribe/Tickets_View.php

```diff
@@ -163,8 +195,10 @@ public function update_tickets() {
 	 * @return void
 	 */
 	public function authorization_redirect() {
+		global $wp_query;
+
 		// When it's not our query we don't care
-		if ( ! tribe_is_event_query() ) {
+		if ( empty( $wp_query->tribe_is_event_query ) ) {
```

Comment:
Why the change here? It feels like we'd be better served to continue using `tribe_is_event_query()` so the value can be filtered.